contrib/golang/filters/http/test/test_data/basic/filter.go

```diff
@@ -109,6 +109,19 @@ func (f *filter) decodeHeaders(header api.RequestHeaderMap, endStream bool) api.
 	if found {
 		md := f.callbacks.StreamInfo().DynamicMetadata()
 		md.Set("filter.go", "foo", "bar")
+		metadata := md.Get("filter.go")
```

Comment:
we'd better cover the empty result, `Get` before `Set`.

Reply:
done, thanks.